lib/puppet/module/task.rb

```diff
@@ -24,6 +61,48 @@ def self.is_tasks_filename?(path)
       return true
     end
 
+    # Copied from TaskInstantiator so we can use the Error classes here
+    # TODO: harmonize on one implementation
```

Comment:
How much work would this be to do now?

Reply:
probably not that much, I'll look into it once I get the puppet-server PR up

Reply:
I'm going to leave this for a bolt ticket